### DISCUSSION TITLE
Is it possible to use this on esp32-idf project without arduino core?

### DISCUSSION THREAD

**Discussion by james-tridentUK** [schreibfaul1/ESP32-audioI2S#777] [Q&A]
I'd like to use this library on an esp32-s3 without referencing the arduino core. Is it possible? Thanks

**Comment by schreibfaul1** [comment]
Some Arduino libraries are required, such as WiFiClient, FS, SD, SPIFFS...

**Comment by fabio-f68** [comment]
It's possible using esp-idf plugin in vscode and using espressif-arduino as managed component. In this mode you can use esp-idf framework and arduino framework together
